service/telemetry.go

```diff
@@ -183,18 +180,18 @@ func (tel *colTelemetry) initOpenCensus(level configtelemetry.Level, instanceID
 }
 
 func (tel *colTelemetry) initOpenTelemetry() (http.Handler, error) {
-	config := otelprometheus.Config{}
+	cfg := otelprometheus.Config{}
```

Comment:
Good change, but unrelated.

Reply:
Reverted.  I believe this was a result of a `config` package previously being imported here and my IDE complaining about the variable shadowing the package name.